tests/CMakeLists.txt

```diff
@@ -53,7 +59,9 @@ add_executable(
 target_link_libraries(
   aarch64-mingw-tests.exe
   GTest::gtest_main
-  dbghelp
```

Comment:
What is/was `dbghelp`?

Reply:
Originally the stack walk printed out a list of functions. It used dbghelp to get symbols and the name of each function. Now the stack walk is just returning the expected count of stack frames.